dfx.json

```
@@ -0,0 +1,26 @@
+{
```

It would be useful to add the DFX version here:
```suggestion
{
  "dfx": "0.24.2",
```